lib/blocklist.c

```diff
@@ -102,8 +102,33 @@ void allowlist_prefix(char *ip, int prefix_len)
 	_add_constraint(addr, prefix_len, ADDR_ALLOWED);
 }
 
+static int is_ip_ipv6(char *ip) {
+	// get length of string including the null-byte
+	int length = strlen(ip) + 1;
+	// don't modify the input string
+	char *new_str = strndup(ip, length);
+	// check if there's a subnet maskChar
+	char *maskChar = strchr(new_str, '/');
```

Comment:
Can you use one variable naming type in this function? We have both `new_str` and `maskChar`? :) I believe that the ZMap style guide says to use `mask_char`.

Reply:
😬 yikes yep, ofc